spec/linter-eslint-spec.js

```diff
@@ -108,6 +110,21 @@ describe('The eslint provider for Linter', () => {
         )
       )
     })
+    it('', () => {
```

Comment:
Ha, whoops, was adding the structure and forgot to fill that in I'll grab that first thing in the morning and merge this.